sampleconfig/core.yaml

```diff
@@ -369,11 +371,11 @@ peer:
 
     # Delivery service related config
     deliveryclient:
-        # Enables this peer to disseminate blocks it pulled from the ordering service
-        # via gossip.
+        # Enables this peer to disseminate blocks it pulls from the ordering service
+        # to other peers in the same organization via gossip.
         # Note that 'gossip.state.enabled' controls point to point block replication
         # of blocks committed in the past.
-        blockGossipEnabled: true
+        blockGossipEnabled: false
```

Comment:
why do you even need to put it false here? maybe comment it out?

Reply:
> why do you even need to put it false here? maybe comment it out?

If it is not set it defaults to true, see https://github.com/hyperledger/fabric/blob/main/core/deliverservice/config.go#L133-L138

We want it to default to false. It will be more consistent for users and maintainers if the logic stays the same across release branches, with just a simple default value change.

Reply:
Maybe like for bft just ban it here https://github.com/hyperledger/fabric/blob/main/core/deliverservice/deliveryclient.go#L253-L255

Reply:
> We want it to default to false. It will be more consistent for users and maintainers if the logic stays the same across release branches, with just a simple default value change that we can mention in the v3.0 release notes.

Agreed